bedrock/foundation/templates/foundation/annualreport/2011.html

```diff
@@ -11,6 +11,7 @@
 {% block body_class %}sand{% endblock %}
 
 {% block extrahead %}
+  {% stylesheet 'tabzilla-static' %}
```

Comment:
Could we add `tabzilla-static.less` to the `annual_2011` bundle instead of including two separate stylesheets?